sw/device/silicon_creator/rom_ext/rom_ext.c

```diff
@@ -858,22 +1039,38 @@ static rom_error_t rom_ext_try_next_stage(boot_data_t *boot_data,
   return error;
 }
 
-static rom_error_t rom_ext_start(boot_data_t *boot_data, boot_log_t *boot_log) {
-  HARDENED_RETURN_IF_ERROR(rom_ext_init(boot_data));
-  const manifest_t *self = rom_ext_manifest();
-  dbg_printf("Starting ROM_EXT %u.%u\r\n", self->version_major,
-             self->version_minor);
+OT_WARN_UNUSED_RESULT
```

Comment:
you dont need to mark OT_WARN_UNUSED_RESULT twice, once in the prototype is fine